packages/launch-config/src/types/types.ts

```diff
@@ -60,6 +59,17 @@ export interface LaunchConfigInfo {
     filePath: string;
 }
 
+export enum DatasourceType {
```

Comment:
Please consider renaming this to something specific to launch config as it could be confused with the odata-service-inquirer datasource type which is a different concept. I'd suggest something like: 
```
type startScriptType = 'cds' | 'mockServer'
```

I cant find any reference to 'odataServiceUrl' so I guess its not needed?